plantuml2mysql.py

```diff
@@ -71,23 +74,25 @@ def main():
             uml = False
             continue
         if not uml:
-             continue
+             continue     
         if l.startswith("class"):
             table = True; field = False
             primary = []; index = ""
             # Table names are quoted and lower cased to avoid conflict with a mySQL reserved word
-            print("CREATE TABLE IF NOT EXISTS `" + i[1].lower() + "` (")
+            print("CREATE TABLE stg." + i[1].lower() + " (")
```

Comment:
Don't understand the syntax here. What is this `stg` prefix?

Reply:
stg is just the schema or the user name in which the table is being created. This was done as we have two layers of tables with same name, same like stg and pub schema in data warehouse systems.